### DISCUSSION TITLE
Prompt configs

### DISCUSSION THREAD

**Discussion by rwmitchell** [romkatv/powerlevel10k#2577] [General]
P10K offers a lot of customization options as well as supporting custom defined prompt segments.

This could be a good place to share interesting configurations and/or custom segments.
Would people be interested in sharing things they think they've done that is clever / interesting?

**Comment by pkazmier** [comment]
I would love to see a gallery of different prompts made with p10k. I’m getting tired of my current prompt (rainbow). And, I’ve started looking around for inspiration, but haven’t found much. So, yes, it would be great to have a gallery of what others have done. We need more styles / looks for p10k.
